.github/workflows/ci.yml

```diff
@@ -34,9 +34,20 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        php: ['7.3', '7.4', '8.0', '8.1']
+        php: ['5.4', '5.5', '5.6', '7.0', '7.1', '7.2', '7.3', '7.4', '8.0', '8.1']
+        dependency-versions: ['highest']
 
-    name: PHP ${{ matrix.php }}
+        include:
+          - php: '7.3'
+            dependency-versions: 'lowest'
+          - php: '7.4'
+            dependency-versions: 'lowest'
+          - php: '8.0'
+            dependency-versions: 'lowest'
```

Comment:
No need to have lowest jobs for each PHP version. We can simplify the matrix.

Reply:
Not sure I understand. Or do you mean to say, you don't want to test against `lowest` ?

The `lowest` runs are only relevant to test the actual polyfill trait in combination with PHPUnit 9.1+ and the PHP versions supported by that.
Running against `lowest` for the availabiity test would end up testing the PHPUnit native implementation across versions, which IMO is not the task of this project.

Reply:
We don't need to have 4 jobs running against `lowest`. One of them is enough.